freud/environment.pyx

```diff
@@ -681,8 +727,14 @@ cdef class EnvironmentMotifMatch(_MatchEnv):
 
     def compute(self, system, motif, threshold, neighbors=None,
                 registration=False):
-        r"""Determine clusters of particles that match the motif provided by
-        motif.
+        r"""Determine which particles' local environment matches given environment motif.
```

Comment:
```suggestion
        r"""Determine which particles have local environments matching the given motif.
```

Reply:
I forgot to check with you. It seems like `point_environments` are also zero-padded. I added a few words in the documentation to warn user about it, but we haven't discussed whether we'll remove those zeros as well

Reply:
Just as another point of clarification, point_environments is the set of all environments of every single particle in the system, rather than the set of average environments of clusters of particles (which is what cluster_environments is). I think that whatever is being done for cluster_environments in terms of zero-padding should also be done for point_environments, since cluster_environments is just built via averages of point_environments.

Reply:
agreed! thanks for putting it so succinctly @vyasr !